### ISSUE TITLE
Axes don't resolve independently in facet charts

### ISSUE THREAD

**Issue by hannahburkhardt** [vega/vega-lite#7067]
Apologies if this is due to user error. I have a chart with dual y-axes which I would like to facet. However, when I do so, the axes no longer resolve independently.
I'm currently manually faceting the chart (using vconcat), which allows the scales to resolve correctly, but I lose the nice facet and axis labels.

Here is a minimal example using the cars dataset:

[Dual axis chart](https://vega.github.io/editor/#/url/vega-lite/N4Igxg9gdgZglgcxALlANzgUwO4tJKAFzigFcJSBnAdTgBNCALFAFgAY2AacaYsiygAlMiRoRQBmDgF9p3OgENCCvCFIAnADYoQYwgAdKyAPTG0mBAoB0COE1IAjK3AhmLCgLSLllTIUrG3grGYArqlFYAVpTQIHIgmgoAnpjqKADaoAC2YQDWOpokmCDcmFCQdCRIqCAAHqqESfrFyCAAjqQKRHZKcOYlIPCYmnQ6ALJwmpiUAPrN6jMA4gqamrHxSQ1NLe2d3crE-dxDIzrUIghiMyQzmg6UcfHY9EwoAKxsctl5BUUDZRUqqp6jVGs0dB0usQDn1iscsKdWhMprN5ksVmsoHFuJtQdsIXtob0joMEaNWoIIOFMPoINhUo9uM8GMxkB9pABdbjqaYQTT9GqUUJTVS4kAkOg0sqSoiMkAAEiFjEwOR0ekMJjclhsdkYjmcriVKuC5ksHkKhEwZhYVgAHFYAIxRGJY6RAA) - axes resolve independently

[Faceted dual axis chart](https://vega.github.io/editor/#/url/vega-lite/N4Igxg9gdgZglgcxALlANzgUwO4tJKAFzigFcJSBnAdTgBNCALFAFgAY2AacaYsiygAlMiRoRQBmDgF9p3OgENCCvCFIAnADYoQYwgAdKyAPTG0mBAoB0COE1IAjK3AhmLCgLSLllTIUrG3grGYArqlFYAVpTQIHIgMApgfqqQmqQAtlCqhACe+pg6UBAZJAra3PCYmnQ6APLqiCRx8ZQFYKqaCrmY6igA2qAZYQDWOpokhdyYUJB0JEioIAAeOfmFyCAAjqQKRHZKcOYglVg1OgCycJqYlAD6Bep3AOLlmrHxuWsFOjt7xMpiMdTtVaptqCIEGI7iQ7poHJQWtxsPQmCgAKxsORDUbjSYnEAzOYLVSrJZ5H6bP77QFHKYJM5gkBXG73R4vN4fbhfcnrX67GmHYEM0E6QQQcKYfQQbC9JEgFEMZjITHSAC68XUtwgmmOS0ooRuqh5IBIdClM3NRHlABIDYxMMMdHpDCY3JYbHZGI5nK57Y7guZLB4JoRMGYWFYABxWACMURi2WkQA) - axes do not resolve independently

If this is indeed a bug - is there a workaround that allows me to add facet labels (headers) manually?
Thanks!

**Comment by hannahburkhardt** [comment]
Correct- I believe the specification you suggested (which is affected by the bug) is what I want. I think I tried that one and decided I must have been doing it wrong because I got that error.
Is there a way to add facet labels (or some other kind of subtitle) to individual charts? When faceting manually by filtering the data and concatenating the resulting subcharts, I can add titles to each subchart plus another separate title to the ConcatChart, but can’t seem to apply separate styles.

**Comment by domoritz** [comment]
What do you mean by separate styles? Can you post a link to what you have for now and describe the change you would want?

**Comment by hannahburkhardt** [comment]
Sure, sorry for the delay.

[Here's what I have now.](https://vega.github.io/editor/#/url/vega-lite/N4Igxg9gdgZglgcxALlANzgUwO4tJKAFzigFcJSBnAdTgBNCALFAFgAY2AacaYsiygAlMiRoRQBmDgF9uxQgBtMeEAEMoYRhABOKEAFt6dJSG4xeAZTgAvZcgCM96bJCMCYVeOQBtUAtUAnpi6PqD6qtoA1noKJMrcmBoQdCRIqCAAHiqEAQAOdiAAjqTq8p5waPEg8JgKdHoAsnBKlAD6+dqtAOKqCgrQIC4B2XkFxaVwhOWVptVYdXrUIghirSStCgBGlIMuhNrqlOba+ii+cwqEwXp0nqT6AHQA8tqIJAC87wA6IACqFgBBH6DAC6sjCEWiyBAsSgVUSkBSUDSoCy6Ry+T04yIk2mVRqC2hTRa7WC3V6-Sgg24w3RoyxJRxU2IMzM83q0MEOkomFyEGw12ccgOUCOOlOoQuVxCIFuhHuz1eCA+3z+gOB0jBIO4ctUKlI2gUejEhFylGQAHoLZUEKoHsqmKRNg84BBrZhbQBaXU8wiUC26i0ebSUB4AK0oAxc2kwkYUM3SlA8JnStJAJDovMSmaIuzkkxTaoBs2w9CYKAArGxwTDAtdJeEojE4rMEclUio0aAMWNGWUWfj2Y1mrHSZ0en0ozSRpjodj+xVB7UOSAlqJCGsoBttnmQPtDscJed4Jd67K7o8Xm8oJ8fgBRA0QfIasGcCFN6Gw+FJJEozIz3sJmZRdZgJFdiVHDpyUnKkhgAhkgLxUCh05bleX5QU9hFMUTjOUAT2lG4L0Va9bxAB9tCfTAX01HVPD1dIDSNaETTNS13Vte1JkYJ0XTdG1VG9ejfX9QNg1DCMpxAGM4wTUAk16OxQDTDMsygHNxCFPcCwKCiqJLMtmGQKsa38IIQnORsoRhFsEh-Dt0i7Pd6TnPtcQHZDl2HEkoInSlqRAWluxcoo3OA1k5i86E1xWDd1i2HYtP3UVDzwqUzzlBUr2VG9VQAKVUXJ1Bot8DEhZs4VbezkU7eDXMQjy2SikAILaXyKSkoLnNnUKGpAprCRALkQ3QgVdCS7DUslAiMuI7KVR+AqiqgGjtXPKZ9UNY1CFNc0rQErjHWdV0OMEn1MD9AN6KDCIJMjWDuBkiB4yUkAFJTZS9FU-J1MSTS9h0vQlvUAyGCMky1vkQt5qpbgABIk0YTBwm23b2IOh0eOOt1EeR1RTs9WIrmtFgHgADgeexw3uwYgA) I've manually concatenated 3 filtered charts to simulate faceting. For each subchart, I add a title with the filter attribute. For the concatchart, I add a title with the column name.

I'd like to make the titles of the 3 filtered charts (USA, Europe, Japan) non-bold, so they look like facet labels, and the top-level title bold, so it looks like an axis label. However, I can only attach a title config to the overall chart, which affects both the "Origin" and facet titles. 

One solution would be to allow separate title configs for the individual subcharts. Another might be to have two levels of titles (e.g. a subtitle) so I can use the subtitle style for the fake facet labels.

**Comment by domoritz** [comment]
You can do this:

```
...
      "title": {
        "text": "USA",
        "fontWeight": "normal"
      },
...
```

**Comment by hannahburkhardt** [comment]
@domoritz Regarding the font weight, this is perfect for what I was trying to do. Thank you!
